Makefile

```diff
@@ -1,4 +1,6 @@
 ### Makefile for tiflow
+include Makefile.engine
```

Comment:
Better to make `include` statement together. L#108

Reply:
Fix it.